ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterManager.java

```diff
@@ -174,6 +176,11 @@ public byte getLocalDatacenterId() {
     return localDatacenterId;
   }
 
+  @Override
+  public String getDatacenterName(byte id) {
+    return dcIdToDcName.get(id);
```

Comment:
same here - can be -1. This should be able to handle and all callers should be able to handle the returned `String` being `null`